src/Razor/test/Microsoft.AspNetCore.Razor.Tools.Test/Microsoft.AspNetCore.Razor.Tools.Test.csproj

```diff
@@ -1,7 +1,7 @@
 ﻿<Project Sdk="Microsoft.NET.Sdk">
 
   <PropertyGroup>
-    <TargetFramework>$(DefaultNetCoreTargetFramework)</TargetFramework>
+    <TargetFramework>netcoreapp5.0</TargetFramework>
```

Comment:
Why?

Reply:
I asked about it offline in an email, but specifically, I thought it might have cause the error:
```
##[error]src\Razor\test\Microsoft.AspNetCore.Razor.Tools.Test\Microsoft.AspNetCore.Razor.Tools.Test.csproj(0,0): error NU1201: (NETCORE_ENGINEERING_TELEMETRY=Restore) Project rzc is not compatible with net50 (.NETFramework,Version=v5.0). Project rzc supports: netcoreapp3.0 (.NETCoreApp,Version=v3.0)
```

seen here: https://dev.azure.com/dnceng/public/_build/results?buildId=637724&view=logs&j=efda9e5f-cf98-536e-4181-9b6c13ac35b3&t=0e3ef6f0-bd22-5e7f-5c24-f1ef533cd236&l=62

Reply:
Turns out this is probably fixed by changing to dotnet msbuild.